tb_plugin/torch_tb_profiler/profiler/trace.py

```diff
@@ -124,8 +122,14 @@ def create_trace_event(event):
         name = event.get("name")
         if name and name.startswith("ProfilerStep#"):
             return ProfilerStepEvent(event)
-
-    if event_type is not None:
-        return TraceEvent(event_type, event)
+        else:
+            return OperatorEvent(event_type, event)
+    elif event_type == EventTypes.PYTHON:
```

Comment:
If change EventTypeMap['Python'] = EventTypes.OPERATOR, then this elif can be removed.